src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1897,13 +1897,13 @@
   <Node Name="BoundInterpolatedStringExpression" Base="BoundExpression" HasValidate="true">
     <!-- Type is required for this node type; may not be null -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
-    <Field Name="Contents" Type="ImmutableArray(Of BoundNode)"/>
+    <Field Name="Contents" Type="ImmutableArray(Of BoundExpression)"/>
     <Field Name="Binder" Type="Binder" Null="disallow" />
   </Node>
   
   <!-- There is no BoundInterpolatedStringText because we use BoundLiteral of type string for this purpose. -->
   
-  <Node Name="BoundInterpolation" Base="BoundNode">
+  <Node Name="BoundInterpolation" Base="BoundExpression" HasValidate="true">
```

Comment:
Do we still want consistency between C# and VB here? If so, we should change BoundInterpolation for C# to be a BoundNode.